src/bloxone_client/api_client.py

```diff
@@ -184,7 +190,17 @@ def param_serialize(self,
 
         # header parameters
         header_params = header_params or {}
-        header_params.update(self.default_headers)
+        headers = {
+            header_client: self.configuration.client_name,
+            header_sdk: sdk_identifier,
+            'Authorization': 'f"Token {self.configuration.api_key}"',
+            "Content-Type": "application/json",
+            "Accept": "application/json",
+        }
+        #header_params = {**headers, **header_params, **self.default_headers}
```

Comment:
Are we planning to support python version less than 3.5 ? 
If not , we can this shorthand to combine all header params in order .

Reply:
3.5 is EOL, so no need to support. https://devguide.python.org/versions/#supported-versions

Reply:
used the shorthand here !